ComponentKit/HostingView/CKComponentRootView.m

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import "CKComponentRootView.h"
+#import "CKComponentRootViewInternal.h"
+
+#import "CKAssert.h"
+
+@implementation CKComponentRootView
+
+static NSMutableArray *hitTestHooks;
```

Comment:
What other scope would you suggest? It would be tedious to require you to pass the hit test hooks into every data source or component hosting view in the app; in an app as large as Facebook's, there's no easy way to require people to do it.